docs/yaml.md

```diff
@@ -271,3 +271,4 @@ binds:
  - /var:/var:rshared,rbind
 rootfsPropagation: shared
 ```
+
```

Comment:
can you remove this extra line!

Reply:
But you got a line for free! Why give it up?

OK, sure.